src/main/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperations.scala

```diff
@@ -666,6 +666,53 @@ object TezosDatabaseOperations extends LazyLogging {
     }
   }
 
+  /** Updates an account by id, marking it with a flag.
+    * @param id the account identifier on database (corresponds to the pkh)
+    * @return the action to run against the db, which will tell how many rows where touched
+    */
+  def activateAccount(id: String): DBIO[Int] =
+    Tables.Accounts
+      .findBy(_.accountId)
+      .extract(id)
+      .map(_.isActivated)
+      .update(true)
+
+  /** Updates an account history by id and asof, marking it with a flag.
+    * @param id the account identifier on database (corresponds to the pkh)
+    * @param asof which flag to set "on"
+    * @return the action to run against the db, which will tell how many rows where touched
+    */
+  def activateAccountHistory(ids: List[String], asof: java.sql.Timestamp): DBIO[Int] =
```

Comment:
Styl-ish: possibly `activateAccountInHistory`? 😄

Reply:
removed this method because after change in logic it was no longer of use